### ISSUE TITLE
Cannot parse font shorthand error when "scales/font-families" rule is added

### ISSUE THREAD

**Issue by siarheiyelin** [signal-noise/stylelint-scales#69]
> Clearly describe the bug
Cannot parse font shorthand error when "scales/font-families" rule is added.

> Which rule, if any, is the bug related to?
 'scales/font-families': ['Roboto'],

> What CSS is needed to reproduce the bug?
```css
.select {
	&::after {
		font: 12px/1;
	}
}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
module.exports = {
    plugins: [
        '@signal-noise/stylelint-scales'
    ],
    rules: {
        'scales/font-families': ['Roboto'],
    }
}
```

> Which version of stylelint are you using?

"13.12.0"

> Which version of stylelint-scales are you using?

"2.0.1"

> How are you running stylelint: CLI?

CLI
"npx stylelint test.scss"

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

It's related to SCSS

> What did you expect to happen?

No parse error.

> What actually happened (e.g. what warnings or errors did you get)?

Cannot parse font shorthand error is thrown

```shell
test.scss
 3:3  ×  Cannot parse font shorthand   parseError
```

**Comment by jeddy3** [comment]
@siarheiyelin Thanks for the report and for using the template.

The rule is complaining because the `font` shorthand [must include a `font-family` part](https://developer.mozilla.org/en-US/docs/Web/CSS/font#syntax) to be valid. The parser fails to parse the shorthand if it doesn't.

Is `font: 12px/1;` valid in SCSS? If it is, we can avoid parsing the shorthand if the value doesn't contain spaces.
